packages/frontend/editor-ui/src/features/credentials/components/CredentialPicker/CredentialPicker.vue

```diff
@@ -18,6 +18,7 @@ const props = defineProps<{
 	appName: string;
 	credentialType: string;
 	selectedCredentialId: string | null;
+	showAll?: boolean;
```

Comment:
An inverted flag `personalOnly` (or event filter function) would make this component more independent from its usages context

Reply:
Inverted this to personalONly. I kept the filtering here, but made it actually support global credentials, that support was missing on the template setup widget.